db/migrate/20190409161747_add_confirmable_to_admin.rb

```diff
@@ -0,0 +1,15 @@
+class AddConfirmableToAdmin < ActiveRecord::Migration[5.2]
+  # Note: You can't use change, as User.update_all will fail in the down migration
```

Comment:
That's interesting. Why ?

Reply:
Did you investigate a bit more ?

Reply:
Nope... the fine people at _plataformatec_ wrote the migration like that in their wiki pages [here](https://github.com/plataformatec/devise/wiki/How-To:-Add-:confirmable-to-Users), and I just went with it without a thought 😉

Reply:
It's because they do a `User.update_all confirmed_at: DateTime.now` (and you don't ;) )